save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/kubernetes/KubernetesManager.kt

```diff
@@ -40,9 +39,11 @@ class KubernetesManager(
     override fun create(executionId: Long,
                         configuration: DockerService.RunConfiguration<PersistentVolumeId>,
                         replicas: Int,
-                        workingDir: String,
     ): List<String> {
-        val (baseImageTag, agentRunCmd, pvId) = configuration
```

Comment:
Destructive declaration may be okay in this case, but whatever you want

Reply:
It was okay when the class had three parameters, but with five of them it became easier to confuse them.